### PR TITLE
PFW-1298: Prusa Firmware build environment v1.0.7 for `build.sh`

### PR DESCRIPTION
**Pull request by 3d-gussner**
Dependencies:
- [x] https://github.com/prusa3d/Arduino_Boards/pull/16 has been merged

Changes:
- Updated to Arduino-IDE 1.8.19
- Updated to [Ardunio_boards 1.0.5](https://github.com/prusa3d/Arduino_Boards/tree/devel)

ToDo after merge:

- [x] Create a tag `1.0.7-Linux64`
- [x] Create a release including `PF-build-env-Linux64-1.0.7.zip`

### CODE REVIEW

File: hardware/PrusaResearch/avr/cores/prusa_einsy_rambo/USBCore.cpp [hunk 1]
```diff
@@ -819,12 +817,6 @@ void USBDevice_::attach()
 	UDIEN = (1<<EORSTE) | (1<<SOFE) | (1<<SUSPE);	// Enable interrupts for EOR (End of Reset), SOF (start of frame) and SUSPEND
 	
 	TX_RX_LED_INIT;
-
-#if MAGIC_KEY_POS != (RAMEND-1)
```
Comment (DRracer) [inline hardware/PrusaResearch/avr/cores/prusa_einsy_rambo/USBCore.cpp:823]: why was this part removed?

File: hardware/PrusaResearch/avr/cores/prusa_einsy_rambo/CDC.cpp [hunk 2]
```diff
@@ -34,14 +34,14 @@ typedef struct
...
-bool _updatedLUFAbootloader = false;
+static u8 wdtcsr_save;
 
 #define WEAK __attribute__ ((weak))
 
 extern const CDCDescriptor _cdcInterface PROGMEM;
 const CDCDescriptor _cdcInterface =
 {
-	D_IAD(0,2,CDC_COMMUNICATION_INTERFACE_CLASS,CDC_ABSTRACT_CONTROL_MODEL,1),
+	D_IAD(0,2,CDC_COMMUNICATION_INTERFACE_CLASS,CDC_ABSTRACT_CONTROL_MODEL,0),
```
Comment (DRracer) [inline hardware/PrusaResearch/avr/cores/prusa_einsy_rambo/CDC.cpp:44]: Why changing `1->0`? Does it influence the USB-CDC communication in some important way?

File: hardware/PrusaResearch/avr/cores/prusa_einsy_rambo/USBCore.cpp (hunk 1 above)
Comment (3d-gussner) [reply hardware/PrusaResearch/avr/cores/prusa_einsy_rambo/USBCore.cpp:823]: This is done with Arduino IDE 1.8.19

File: hardware/PrusaResearch/avr/cores/prusa_einsy_rambo/CDC.cpp (hunk 2 above)
Comment (3d-gussner) [reply hardware/PrusaResearch/avr/cores/prusa_einsy_rambo/CDC.cpp:44]: Again it is done by Arduino IDE 1.8.19

Comment (DRracer) [reply hardware/PrusaResearch/avr/cores/prusa_einsy_rambo/CDC.cpp:44]: resolved in https://github.com/prusa3d/Arduino_Boards/pull/16

File: hardware/PrusaResearch/avr/cores/prusa_einsy_rambo/USBCore.cpp (hunk 1 above)
Comment (DRracer) [reply hardware/PrusaResearch/avr/cores/prusa_einsy_rambo/USBCore.cpp:823]: ok

**Review by DRracer — APPROVED** [review]
approving
